components/brave_rewards/resources/page/promos/index.tsx

```diff
@@ -20,41 +22,55 @@ export interface Promo {
 export const getActivePromos = (rewardsData: Rewards.State) => {
   let promos = ['tap-network']
 
+  if (rewardsData) {
+    let wallet = rewardsData.externalWallet
+    if (wallet && wallet.address.length > 0 && wallet.status === 2) { // WalletStatus::VERIFIED
+      promos.unshift('uphold')
+    }
+  }
+
   return promos
 }
 
-const openLink = (type: PromoType, rewardsData: Rewards.State) => {
-  let url
+const openLink = (type: PromoType) => {
```

Comment:
Maybe change the name of this function since it's no longer opening a link? Or maybe it just needs a `get` in front of it?

Reply:
changed to `getLink`